source/connect/tls-ssl-connection.txt

```diff
@@ -11,3 +11,106 @@ TLS/SSL Connection Tab
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`TLS / SSL` tab allows you to connect deployments via TLS / SSL.
+For more information on :guilabel:`TLS / SSL`, see :manual:`TLS Options 
+</reference/connection-string/#tls-options>`
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`TLS / SSL` tab.
+
+      You can either use a ``Default`` connection or to set the TLS / SSL 
+      connection ``On`` or ``Off``. 
+
+      .. list-table::
+         :header-rows: 1 
+         :widths: 50 50
+      
+         * - Option
+           - Description
+
+         * - Default
+           - The ``Default`` option does not use the ``TLS / SSL`` connection option. 
+             When using the ``Default`` option, you can specify additional certificate 
+             options. To see more on the additional options available, see 
+             :ref:`<additional-tls>`.
+      
+         * - On
+           - The ``On`` option is enabled when using a :manual:`DNS seedlist (SRV) 
```

Comment:
Because the `On` option is "enabled" for all connection strings, maybe we should tweak this to say "You can choose the `On` option when using a...."